packages/ui/src/PasswordField.tsx

```diff
@@ -38,7 +38,7 @@ export type PasswordFieldExtraProps = {
   withIcon?: boolean
   visibleRef?: Ref<VisibleRef>
 } & DataTestProp &
-  Pick<InputHTMLAttributes<HTMLInputElement>, 'autoFocus' | 'disabled' | 'autoComplete' | 'required' | 'placeholder'>
+  Pick<InputHTMLAttributes<HTMLInputElement>, 'id' | 'autoFocus' | 'disabled' | 'autoComplete' | 'required' | 'placeholder'>
```

Comment:
That `id` is passed to `TextField` that generates its own `id`. Should we use the external `id` that we pass (if present) instead of the autogenerated one?

Reply:
Well, the autogenerated `id` is overloaded by `{...htmlAttrs}`, so it works as expected.

Reply:
But maybe that's not entirely clear on first look. Maybe we can do it like this in `TextField`?
```
id={htmlAttrs.id ?? autoId}
```
What do you think?

Reply:
@jozefhruska the autogenerated id is overridden only for the `input`, but not for `Help`, `Error` and `Label`

Reply:
Good catch! I added an overload to `TextField`. Sorry, this is a quick change for Cloud release... Guess I rushed things a bit 🙈